enhancements/declarative-index-config.md

```diff
@@ -865,34 +865,22 @@ $ tree
 ├── etcd.json
 ├── amqstreams.json
 ```
-#### Editing a package inside an index
+#### Creating new indexes using the config files/Editing a package inside an existing index
 
-A new sub-command `update` will be introduced under `opm index`, which will take a folder containing json representations of packages as it's input along with an existing index image tag, and will replace the edited config files in the index container. 
+A new sub-command `build` will be introduced under `opm index`, which will take a folder containing json representations of packages as it's input, and will build a container image with the folder and the `opm` binary in it. 
 
 ```bash
 $ opm index inspect --image=quay.io/some-namespace/my-index 
```

Comment:
same as a couple of other comments; the image should be given as an operand, not an option.

There are a few more things I noticed about the current design of `inspect` that I might as well mention:

- `json` is a misleading option-arg for `--out`,  since the option actually denotes the index target format. It's conceivable that there is some index format that's _not_ a directory filled with package files, but which is still serialized as JSON; e.g. a single JSON blob containing all packages.
- "index image" is actually a format unto itself; i.e. we should be able to inspect any two index images that serve identical content over gRPC and get the same output, regardless of how either image is backed.

Reply:
The `--out` options is supposed to only give the users the ability to read the output of `inspect` in a format they're most comfortable with (json/yaml...etc). We can interchange between a directory filled with package files and a single JSON blob but still provide a way for users to choose between how they are most comfortable reading that output.

Reply:
Got it. Then you need an additional option to specify the target format.